tests/unit/gapic/recommendationengine_v1beta1/test_prediction_service.py

```diff
@@ -22,46 +22,40 @@
 except ImportError:  # pragma: NO COVER
     import mock
 
-import grpc
-from grpc.experimental import aio
 from collections.abc import Iterable
-from google.protobuf import json_format
 import json
 import math
-import pytest
-from proto.marshal.rules.dates import DurationRule, TimestampRule
-from proto.marshal.rules import wrappers
-from requests import Response
-from requests import Request, PreparedRequest
-from requests.sessions import Session
-from google.protobuf import json_format
 
+from google.api_core import gapic_v1, grpc_helpers, grpc_helpers_async, path_template
```

Comment:
when does it include multiple imports in one line like here, and when in multiple lines with parens?

Reply:
I believe this is part of `black` which automatically formats code which exceeds the line length limit. In the future we will remove the need to run `black` when we remove post processing altogether. 

https://github.com/googleapis/python-recommendations-ai/blob/19719d1ba6c1bed8c5552ab3729533804ab55e20/noxfile.py#L117-L120